Cloudflare setup issues - Workers subscription required, enable R2 in dashboard

Hi again Josh :-)

I had a couple more issues setting up a new project that included all the features - D1, R2, Queue and Durable Object.

I got an error saying:

```
✘ [ERROR] A request to the Cloudflare API (/accounts/xxx/r2/buckets) failed.

  Please enable R2 through the Cloudflare Dashboard. [code: 10042]
```

That was straightforward enough to fix myself, but might be worth adding a note in the setup instructions to do that before starting.

But the other error from the Workers SDK was more vague:

```
✘ [ERROR] A request to the Cloudflare API (/accounts/xxx/workers/queues) failed.

  workers.api.error.not_entitled [code: 10015]
```

I ended up working out that I needed a paid Workers subscription (the $5 plan) in order to use the Queues and Durable Object features. I didn't see that mentioned in the docs. As soon as I set up the paid subscription, the command worked properly.

Thanks Josh, hope this is helpful.

--- a/.changeset/violet-kangaroos-grow.md
+++ b/.changeset/violet-kangaroos-grow.md
@@ -1,0 +1,5 @@
+---
+"superflare": patch
+---
+
+Add helpers to get people to check their Cloudflare Queue, R2, etc settings

--- a/packages/superflare/bin/superflare.js
+++ b/packages/superflare/bin/superflare.js
@@ -4,7 +4,7 @@
 const { spawn } = require("child_process");
 const path = require("path");
 
-const MIN_NODE_VERSION = "16.13.0";
+const MIN_NODE_VERSION = "18.0.0";
 
 let superflareProcess;
 

--- a/packages/superflare/cli/new.ts
+++ b/packages/superflare/cli/new.ts
@@ -8,6 +8,7 @@ import {
   outro,
   spinner,
   text,
+  note,
 } from "@clack/prompts";
 import { cp, mkdtemp, readFile, rmdir, writeFile } from "fs/promises";
 import { join, normalize } from "path";
@@ -75,6 +76,14 @@ export async function newHandler(
   }
 
   s.stop("Everything looks good!");
+
+  note(
+    "Before using R2, Queues, and Durable objects,\n" +
+      "make sure you've enabled them in the Cloudflare Dashboard.\n" +
+      "https://dash.cloudflare.com/\n" +
+      "Otherwise, the following commands might fail! 😬",
+    "👋 Heads-up:"
+  );
 
   let path = "";
 
@@ -242,9 +251,11 @@ Do you want to continue?`;
   const appKey = randomBytes(256).toString("base64");
   await setSecret("APP_KEY", appKey, path);
 
-  s.stop("Done creating resources:");
+  s.stop("Done creating resources!");
 
-  results.forEach((result) => logger.log(result.message));
+  const allResults = results.map((r) => r.message);
+
+  note(allResults.join("\n"), "Here's what we did:");
 
   // TODO: Drop the `--legacy-peer-deps` once the remix template is fixed:
   // @see https://github.com/remix-run/remix/pull/5425

